src/index.js

```diff
@@ -848,6 +865,7 @@ export {
  * @prop {() => TxChain} build
  * @prop {(id: TxOutputId) => Promise<TxInput>} getUtxo
  * @prop {(addr: Address) => Promise<TxInput[]>} getUtxos
+ * @prop {(utxoId: TxOutputId) => Promise<boolean>} hasUtxo;
```

Comment:
Same comment as above: I think it will be cleaner to remove the final `;`

Reply:
done